src/core/TreeBuilder.lua

```diff
@@ -0,0 +1,48 @@
+return function(job_queue)
```

Comment:
GitHub lost its mind, but the only changes are here, line 7, and the return value (now a table)